docs/articles/documentation/guides/basic-guides/debug.md

```diff
@@ -1,43 +1,35 @@
 ---
 layout: docs
-title: Debugging
-permalink: /documentation/test-api/debugging.html
-checked: true
+title: Debug
+permalink: /documentation/guides/basic-guides/debug.html
 ---
-# Debugging
+# Debug
 
 TestCafe allows you to debug server-side test code and test behavior on the client.
 
-* [Debugging Test Code](#debugging-test-code)
+* [Debug Test Code](#debug-test-code)
 * [Client-Side Debugging](#client-side-debugging)
 * [Options Useful for Debugging](#options-useful-for-debugging)
 
-## Debugging Test Code
+## Debug Test Code
 
 You can debug test code in Chrome Developer Tools and popular IDEs. See the following recipes for details.
 
-* [Debug in Chrome Developer Tools](../recipes/debug-in-chrome-dev-tools.md)
-* [Debug in Visual Studio Code](../recipes/debug-in-visual-studio-code.md)
+* [Debug in Chrome Developer Tools](../../recipes/debug-in-chrome-dev-tools.md)
+* [Debug in Visual Studio Code](../../recipes/debug-in-visual-studio-code.md)
 
 ## Client-Side Debugging
 
-TestCafe provides the `t.debug` method that pauses the test and allows you to debug using the browser's developer tools.
+TestCafe provides the [t.debug](../../reference/test-api/testcontroller/debug.md) method that pauses the test and allows you to debug using the browser's developer tools.
```

Comment:
allows you to debug what?

Reply:
Oh, yes, it should read:

_TestCafe provides the t.debug method that pauses the test and allows you to debug it with the browser's developer tools._

Reply:
great